src/NLog/Config/LoggingConfiguration.cs

```diff
@@ -376,7 +392,9 @@ public void AddRuleForOneLevel(LogLevel level, Target target, string loggerNameP
         }
 
         /// <summary>
-        /// Add a rule for all loglevels.
+        /// Add a rule for all levels. The rule will NOT have a name.<br/>
+        /// If a name is required, use <see cref="LoggingRule(string)"/> instead and add the created rule to the
+        /// <see cref="LoggingRules"/> collection of this class.
```

Comment:
The LoggingRule-name is an exotic feature, that is only relevant for special users that wants to special changes to NLog-configuration at runtime. Think normal users will wonder what is the meaning of rule-name? Is a rule-name needed? What happens if no rule-name is provided? I prefer that xml-comments doesn't mention all the stuff that are not assigned.